src/clj/web/user.clj

```diff
@@ -9,6 +9,22 @@
    :isadmin :ismoderator :tournament-organizer
    :special :options :stats :has-api-keys :banned])
 
+(def character-length #(.codePointCount % 0 (count %)))
```

Comment:
Doesn't `count` call String's `length`, which returns the number of code points in the Java String?

Reply:
No, I originally used `count` and it was doing a byte count, not glyph count.

Reply:
hmmmmmmmm. This will probably also throw a reflection warning, so maybe
```suggestion
(defn character-length [^String username]
  (.length username))
```